### PR TITLE
Full Mavenization of the project

### PR DESCRIPTION
**Pull request by javydreamercsw**
All tests pass now except one testWriteToJson() which sems to be legit failure as some fields are reversed in order. It doesn't seem to be related to this changes.

Also updated the ant scripts so everything should work in both systems.

### CODE REVIEW

**Comment by nhnb** [comment]
This pull requests consists of 30000 changed lines and 40 commits. I think there is something wrong with it.

**Comment by javydreamercsw** [comment]
If this is still desired I can create a cleaner one. Just hard to follow up after 8 months.

**Comment by balaur64** [comment]
Yes, please. :)

**Comment by javydreamercsw** [comment]
Here it is: https://github.com/arianne/marauroa/pull/16
